src/pages/Home.jsx

```diff
@@ -1,11 +1,33 @@
-import ApplicationForm from "../components/ApplicationForm";
 import MotivationalQuotes from "../components/MotivationalQuotes";
+import JobList from "../components/JobList";
+import "../index.css";
 
 function Home() {
   return (
-    <div>
-      <MotivationalQuotes />
-      <ApplicationForm />
+    <div className=" bg-softPink flex flex-col items-left justify-start p-5 gap-5">
+      <div className="flex flex-row">
+        <div className="bg-linear-to-r from-gray-50 to-white shadow-md ml-8 text-3xl  font-lora border border-none rounded-3xl p-8 w-1/2 text-left hover:border-2 hover:border-solid hover:border-blue-500">
+          {" "}
+          Hi [user!]
+        </div>
+        <div className="bg-linear-to-r from-gray-50 to-white shadow-md ml-8 text-3xl font-lora border border-none rounded-3xl p-8 w-1/4 text-left hover:border-2 hover:border-solid hover:border-blue-500 hover:underline hover:decoration-blue-500">
+          {" "}
+          Job Tracking
+        </div>
```

Comment:
```suggestion
        <NavLink to={"/jobtracking"}>
          <div className="bg-linear-to-r from-gray-50 to-white shadow-md ml-8 text-3xl font-lora border border-none rounded-3xl p-8 w-1/4 text-left hover:border-2 hover:border-solid hover:border-blue-500 hover:underline hover:decoration-blue-500">
            Job Tracking
          </div>
        <NavLink />
```

Wrap this `div` with a `NavLink` so it can redirect the user to the job tracking page when it is clicked. Same idea with the Resources nav button.

Reply:
@wukellyy  Thank you for your suggestion. I’ll definitely work on this in a later PR, and I’ll check the error in the Motivational Quote section and try to resolve it.